specification/translation/Azure.AI.DocumentTranslation/examples/2024-05-01/operations.json

```diff
@@ -16,7 +16,7 @@
     "statuses": [
       "Succeeded"
     ],
-    "ids": [
+    "translationIds": [
```

Comment:
Example of why this would break with the service...this is an example of what the service would return so unless the service changes this (which would be a breaking change) the renames need to be done in client.tsp

Reply:
Updated